indico/web/templates/forms/colorswitch_widget.html

```diff
@@ -0,0 +1,22 @@
+{% extends 'forms/base_widget.html' %}
+
+{% block html %}
+    <div class="group">
+        <span class="color-switch">
+            <a class="i-button icon-expand switch-trigger"
+               style="background: #{{ field._value()['background'] }};
+                      color: #{{ field._value()['text'] }} !important">
+                {%- trans %}Choose a color{% endtrans -%}
+            </a>
+        </span>
+        <input type="hidden" name="{{ field.name }}" value={{ field._value() | tojson }}>
+    </div>
+    <script>
+        $('.color-switch').colorswitch({
```

Comment:
this is way too generic! Always make sure JS in here only selects that specific field (i.e. use the field id or an id containing it).